plugins/SitesManager/Controller.php

```diff
@@ -167,17 +167,23 @@ public function siteWithoutData()
     }
 
     public function siteWithoutDataTabs() {
-        $typeCacheId = 'guessedtype_' . md5($this->site->getMainUrl());
-        $gtmCacheId = 'guessedgtm_' . md5($this->site->getMainUrl());
+        $mainUrl = $this->site->getMainUrl();
+        $typeCacheId = 'guessedtype_' . md5($mainUrl);
+        $gtmCacheId = 'guessedgtm_' . md5($mainUrl);
 
         $siteType = $this->cache->fetch($typeCacheId);
         $gtmUsed = $this->cache->fetch($gtmCacheId);
 
         if (!$siteType) {
             try {
-                $response = Http::sendHttpRequest($this->site->getMainUrl(), 5, null, null, 0, false, false, true);
-            } catch (Exception $e) {
                 $response = false;
+                $parsedUrl = parse_url($mainUrl);
+
+                // do not try to determine the site type for localhost or any IP
+                if (!empty($parsedUrl['host']) && $parsedUrl['host'] !== 'localhost' && !filter_var($parsedUrl['host'], FILTER_VALIDATE_IP)) {
+                    $response = Http::sendHttpRequest($mainUrl, 5, null, null, 0, false, false, true);
+                }
+            } catch (Exception $e) {
```

Comment:
Is it possible to log this exception at INFO level?

Reply:
done

Reply:
done